build.sbt

```diff
@@ -72,6 +85,7 @@ lazy val indexing = project
       "io.verizon.journal"         %% "core"               % journalVersion.value,
       "org.apache.jena"            % "jena-arq"            % jenaVersion,
       "org.apache.jena"            % "jena-querybuilder"   % jenaVersion,
+      "org.apache.jena"            % "jena-querybuilder"   % jenaVersion,
```

Comment:
any reason for this duplication?

Reply:
No. Obtained I think after conflict resolving. Removed.